### PR TITLE
Update metrics

### PR DESCRIPTION
**Pull request by Liuhaai**
1. Update"account", "project", "publisher", "eventtype" tags for event metrics
2. Update "account", "project" tags for publisher metrics
3. Decrease publisher metrics when the publishers are deleted

### CODE REVIEW

File: pkg/modules/publisher/publisher.go [hunk 1]
```diff
@@ -142,7 +145,11 @@ func RemoveBySFID(ctx context.Context, id types.SFID) error {
 			}
 			return nil
 		},
-	).Do()
+	).Do(); err != nil {
+		return err
+	}
+	_publisherMtc.WithLabelValues(acc.AccountID.String(), prj.Name).Dec()
```
[the remark this answers is not in the document]
Comment (CoderZhi) [reply pkg/modules/publisher/publisher.go:151]: @hunshenshi If a publisher is deleted and then added back, will `_publisherMtc` calculate the unique number? If not, the definition of `_publisherMtc` has some problem.

File: cmd/srv-applet-mgr/apis/publisher/del.go [hunk 2]
```diff
@@ -33,12 +33,12 @@ type BatchRemovePublisher struct {
...
+	acc := middleware.MustCurrentAccountFromContext(ctx)
+	ctx, err := acc.WithProjectContextByName(ctx, middleware.MustProjectName(ctx))
 	if err != nil {
 		return nil, err
 	}
-	r.ProjectIDs = []types.SFID{types.MustProjectFromContext(ctx).ProjectID}
-
-	return nil, publisher.Remove(ctx, &r.CondArgs)
+	prj := types.MustProjectFromContext(ctx)
+	r.ProjectIDs = []types.SFID{prj.ProjectID}
```
Comment (CoderZhi) [inline cmd/srv-applet-mgr/apis/publisher/del.go:42]: is `ProjectIDs` in use in db operation? if so, could it be conflict with `prj`. If not, no need to specify its value

File: pkg/modules/publisher/publisher.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (hunshenshi) [reply pkg/modules/publisher/publisher.go:151]: `_publisherMtc` is not calculated with the unique number. `_publisherMtc` records only the times of registered publishers in a project. If a publisher is deleted and then added again, the `_publisherMtc` is 2. `_publisherMtc` represents how many times the registration action has occurred.

[the remark this answers is not in the document]
Comment (Liuhaai) [reply pkg/modules/publisher/publisher.go:151]: > @Liuhaai perhaps we should add a metric for the number of publishers deleted

any benefits for two metrics?

**Comment by CoderZhi** [comment]
@saitofun leave this pr to you

**Comment by saitofun** [comment]
> @saitofun leave this pr to you

👌

**Review by CoderZhi — APPROVED** [review]
The implementation is not ideal, but it works in current circumstance. So, let's merge it first, and fix the implementation in a follow up PR.

**Pull request merged**